File: Iniciante/2176.py
s = str(input())
cont1 = 0

for i in range(len(s)):
  if (s[i] == '1'):
    cont1 += 1

if (cont1%2 == 0):
  print('{}0'.format(s))
else:
  print('{}1'.format(s))

File: Iniciante/2176.js
let s = lines.shift();
let cont1 = 0;

for (let i = 0; i < s.length; i++) {
  if (s[i] == '1') {
    cont1++;
  }
}

if (cont1%2 == 0) {
  console.log(s + '0');
} else {
  console.log(s + '1');
}
